Analysis/PhotonEnergyCorrect/doc/README

```diff
@@ -0,0 +1,28 @@
+This is a processor to make energy corrections to photon PFOs.
```

Comment:
Can you format this to markdown. It's nicely rendered by Github for us

Reply:
done